.github/workflows/4-deploy-to-prod.yml

```diff
@@ -23,7 +23,10 @@ jobs:
 
       - uses: actions/checkout@v3
       - uses: ./.github/actions/setup
+      
       - run: yarn build
+        env:
+          IS_IPFS_BUILD: 1
```

Comment:
The same build is deployed to both IPFS and CF. In order to differentiate, you'd need to either:
- make two separate builds
- use a runtime check on an env var set in CF
- check the origin (ie `isAppUniswapOrg() || isAppUniswapStagingOrg()`)

Reply:
Yeah, sorry, for some reason, not a full diff got shipped. I initially settled on 
> make two separate builds
So, re-build in the latter stage with a flag set. Considered that a bit easier to maintain than two separate build pipelines.